packages/zwave-js/src/lib/controller/Controller.ts

```diff
@@ -1970,6 +1985,21 @@ ${associatedNodes.join(", ")}`,
 			`starting replace failed node process...`,
 		);
 
+		const node = this.nodes.get(nodeId);
+		if (!node) {
+			throw new ZWaveError(
+				`Node ${nodeId} was not found!`,
+				ZWaveErrorCodes.Controller_NodeNotFound,
+			);
+		}
+
+		if (await node.ping()) {
+			throw new ZWaveError(
+				`The node replace process could not be started because the node responded to a ping.`,
+				ZWaveErrorCodes.ReplaceFailedNode_Failed,
+			);
+		}
```

Comment:
Here you use the same code you used previously, could be a function so

Reply:
Different error message. I don't see the benefit in having a method like getNodeOrThrowAndThrowIfPingSuceeded.
The first part, okay.